tinygrad/runtime/ops_gpu.py

```diff
@@ -23,8 +23,8 @@ def render(self, name:str, uops) -> str: return OpenCLRenderer(name, uops)
   def compile(self, src:str) -> bytes:
     program = checked(cl.clCreateProgramWithSource(self.device.context, 1, to_char_p_p([prg_bytes := src.encode()]),
                                                   ctypes.byref(ctypes.c_size_t(len(prg_bytes))), ctypes.byref(status := ctypes.c_int32())), status)
-    status = cl.clBuildProgram(program, 1, ctypes.byref(self.device.device_id), None, cl.clBuildProgram.argtypes[4](), None)
-    if status != 0:
+    build_status: int = cl.clBuildProgram(program, 1, ctypes.byref(self.device.device_id), None, cl.clBuildProgram.argtypes[4](), None)
```

Comment:
mypy's `--strict-equality` complains about this because in line 25 `status` is a ctypes.c_int32()

Reply:
Oh, this is actually a bug. Should be `status.value == 0`

Reply:
status was int when i tested though?

Reply:
i tried doing status.value and actually it doesn't exist, the correct way was to do status != 0 (mac)